media/static-bundles.json

```diff
@@ -1595,10 +1595,17 @@
     {
       "files": [
         "js/base/send-to-device.js",
+        "js/base/send-yourself.js",
```

Comment:
There's no need to add this file to the regular `firefox_whatsnew` bundle, it should be removed.